docs/comparisons/testing/property_based_testing.md

```diff
@@ -0,0 +1,26 @@
++++
+title = "Property Based Testing"
++++
+
+# Property Based Testing
+\toc
+
+## Introduction
+Programming and mathematics have always interested similar minds, and tackled related problems. Mathematics has strong formalism in place about the properties of certain objects (scalars, vectors, matrices) under certain operations (addition, multiplication, transposition), or even properties of the operators themselves. Property Based Testing (PBT) is an attempt to use computer programming to test if certain properties hold for certain objects.
+
+## Overview
+There are two main packages in this domain: [PropCheck.jl](https://github.com/Seelengrab/PropCheck.jl), and [Suppositon.jl](https://github.com/Seelengrab/Supposition.jl). Both packages are created by the same person, who has made a comparison page of the two packages in the [FAQ of the Supposition.jl docs](https://seelengrab.github.io/Supposition.jl/stable/faq.html). The general takeaway is that PropCheck.jl is in maintainance mode, and that Supposition.jl should be concidered a superior successor to PropCheck.jl.
```

Comment:
```suggestion
There are two main packages in this domain: [PropCheck.jl](https://github.com/Seelengrab/PropCheck.jl), and [Suppositon.jl](https://github.com/Seelengrab/Supposition.jl). Both packages are created by the same person, who has made a comparison page of the two packages in the [FAQ of the Supposition.jl docs](https://seelengrab.github.io/Supposition.jl/stable/faq.html). The general takeaway is that PropCheck.jl is in maintainance mode, and that Supposition.jl should be considered a successor to PropCheck.jl.
```

There are still some things that PropCheck.jl can do that Supposition.jl can't do, so I wouldn't call it entirely superior (yet) :) There's also a typo here (`concidered` -> `considered`).